pkg/trace/obfuscate/sql.go

```diff
@@ -186,28 +189,38 @@ func obfuscateSQLString(in string) (string, error) {
 		}
 		lastToken = token
 	}
+	if out.Len() == 0 {
+		return "", errors.New("result is empty")
+	}
 	return out.String(), nil
 }
 
 // QuantizeSQL generates resource and sql.query meta for SQL spans
 func (o *Obfuscator) obfuscateSQL(span *pb.Span) {
+	tags := []string{"type:sql"}
```

Comment:
What about the other obfuscation types? Only tracking sql for now?

Reply:
Yeah, we only care about `sql` for now, I just added this to allow slicing by tags in case we decide to track more of them.

Reply:
👍